### PR TITLE
Minor fixes for Lang.php

### PR DESCRIPTION
**Pull request by jdarwood007**
Declare ValueError usage
formatText does not accept 3 parameters.

### CODE REVIEW

File: Sources/Lang.php [hunk 1]
```diff
@@ -326,7 +327,7 @@ public static function load(string $template_name, string $lang = '', bool $fata
 
 			// That couldn't be found!  Log the error, but *try* to continue normally.
 			if (!$found && $fatal) {
-				ErrorHandler::log(self::formatText(self::$txt['theme_language_error'] ?? 'Unable to load the {filename} language file.', ['filename' => $lang . '/' . $template_name], 'template'));
+				ErrorHandler::log(self::formatText(self::$txt['theme_language_error'] ?? 'Unable to load the {filename} language file.', ['filename' => $lang . '/' . $template_name]));
```
Comment (Sesquipedalian) [inline Sources/Lang.php]:
```suggestion
				ErrorHandler::log(self::formatText(self::$txt['theme_language_error'] ?? 'Unable to load the {filename} language file.', ['filename' => $lang . '/' . $template_name]), 'template');
```

**Review by Sesquipedalian — CHANGES_REQUESTED** [review]

File: Sources/Lang.php [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 use SMF\Cache\CacheApi;
 use SMF\Localization\MessageFormatter;
+use ValueError;
```
Comment (Sesquipedalian) [inline Sources/Lang.php]: If we are adding this, then we might as well replace all instances of `\ValueError` with just `ValueError` in the rest of the file.

**Review by Sesquipedalian — CHANGES_REQUESTED** [review]

File: Sources/Lang.php (hunk 2 above)
Comment (jdarwood007) [reply Sources/Lang.php]: I could go either way.  Just an IDE warning while I was looking at it. 

What is the preferred method here for PHP's built-ins?  Use \ or define the use?

Comment (Sesquipedalian) [reply Sources/Lang.php]: I generally prefer using fully qualified names (e.g. `\ValueError`) for built-in classes, for two reasons. 

1. There is less to maintain. Creating aliases of classes in the global namespace in the local namespace via `use` statements accomplishes very little except requiring us to spend more time keeping track of `use` statements.
2. It makes the code clearer. When I see a FQN in the code, I can immediately know that I am dealing with a built-in class, without having to waste any time scrolling to the top of the file to see if there are any unexpected class aliases in play.

All of which is to say that I'd actually prefer to see this `use` statement removed.

Comment (jdarwood007) [reply Sources/Lang.php]: Thats fine with me.  Pick a standard.  So for built-ins,  we use \, for all others we define the use.

**Review by Sesquipedalian — APPROVED** [review]

**Pull request merged**